PayButton/paybutton-server
# [Issue #182] feat: in and out transactions

**Descripition:**
Solves #182 . An outward transaction from some address will have a negative amount.

**Test plan:**
Let `bchaddr` be some bitcoin cash address with IN and OUT transactions. Then:
- Create a Paybutton with the address `bchaddr`, 
- Run `curl -X GET http://localhost:3000/api/transactions/sync/bchaddr` to save the transactions in the database,
- Run `curl -X GET http://localhost:3000/api/transactions/bchaddr | jq` and  see if the transactions are correctly displayed.


Depends on:
- [x] #176 
- [x] #178
